src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/uiTest/QuickAccessParametersTest.kt

```diff
@@ -0,0 +1,270 @@
+            "Search budget", quickAccessParameters.searchBudgetSeparator
+        ),
+        Arguments.of(
+            "Timeouts", quickAccessParameters.timeoutsSeparator
+        ),
+        Arguments.of(
+            "Genetic Algorithm", quickAccessParameters.geneticAlgorithmSeparator
+        )
+    )
+
+    @Order(4)
+    @ParameterizedTest
+    @MethodSource("valueGeneratorForDefaultTooltipsAndTexts")
+    fun testToolTipsOnUIElementLabels(expectedText: String, tooltipLabel: JLabelFixture) {
+        // TODO: add other default tooltips
+        Assertions.assertThat(tooltipLabel.value).isEqualTo(expectedText)
+    }
+
+    private fun valueGeneratorForDefaultTooltipsAndTexts(): Stream<Arguments> = Stream.of(
+        Arguments.of(
+            "Default: Max time", quickAccessParameters.searchBudgetTypeDefaultTooltip
+        ),
+        Arguments.of(
+            "Default: 120 seconds", quickAccessParameters.initializationTimeoutDefaultTooltip
+        ),
+        Arguments.of(
+            "Default: Individuals", quickAccessParameters.populationLimitDefaultTooltip
+        )
+    )
+
+    @Order(5)
+    @Test
+    fun testComboBoxesAreVisible() {
+        quickAccessParameters.getComboBoxes().forEach { it.isShowing }
+    }
+
+    @Order(6)
+    @Test
+    fun testSpinnersAreVisible() {
+        quickAccessParameters.getJSpinners().forEach { Assertions.assertThat(it.isShowing) }
+    }
+
+    @Order(7)
+    @ParameterizedTest
+    @MethodSource("valueGeneratorForSearchBudgetTypeComboBoxChoices")
+    fun testSearchBudgetComboBoxes(choice: String): Unit = with(remoteRobot) {
+        quickAccessParameters.searchBudgetTypeArrow.click()
+        val choices: JListFixture = find(byXpath("//div[@class='JList']"), Duration.ofSeconds(15))
+        choices.clickItem(choice)
+        Assertions.assertThat(quickAccessParameters.searchBudgetTypeComboBox.hasText(choice))
+    }
+
+    private fun valueGeneratorForSearchBudgetTypeComboBoxChoices(): Stream<Arguments> = Stream.of(
+        Arguments.of("Max statements"),
+        Arguments.of("Max tests"),
+        Arguments.of("Max generations"),
+        Arguments.of("Max fitness evaluations"),
+        Arguments.of("Time delta"),
+        Arguments.of("Max time"),
+    )
+
+    @Order(8)
+    @ParameterizedTest
+    @MethodSource("valueGeneratorForPopulationLimitComboBoxChoices")
+    fun testPopulationLimitComboBoxes(choice: String): Unit = with(remoteRobot) {
+        quickAccessParameters.populationLimitArrow.click()
+        val choices: JListFixture = find(byXpath("//div[@class='JList']"), Duration.ofSeconds(15))
+        choices.clickItem(choice)
+        Assertions.assertThat(quickAccessParameters.populationLimitComboBox.hasText(choice))
+    }
+
+    private fun valueGeneratorForPopulationLimitComboBoxChoices(): Stream<Arguments> = Stream.of(
+        Arguments.of("Tests"),
+        Arguments.of("Statements"),
+        Arguments.of("Individuals"),
+    )
+
+    // @Order(9)
```

Comment:
This can be deleted?

Reply:
@bolkhod has already said that. I have uncommented it